core/src/generated.rs

```diff
@@ -0,0 +1,12 @@
+#![allow(clippy)]
+#![allow(non_snake_case)] // because of the generated bindings.
+
+use sys;
+use get_api;
+use super::*;
+
+use std::sync::{Once, ONCE_INIT};
+use std::ops::*;
+use libc;
+
+include!(concat!(env!("OUT_DIR"), "/core_types.rs"));
```

Comment:
This whole file seems out of place? There already is [`gdnative-core/src/generated.rs`](https://github.com/GodotNativeTools/godot-rust/blob/ca9649d6f79dbeded39863fb4df20950521c07f4/gdnative-core/src/generated.rs)

Reply:
Whoops, must have added by accident.